NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -359,8 +359,58 @@ private List<Entity> ConvertComponentPrefabsToEntities(List<PrefabAsset> prefabs
                              counter++,
                              parent);
 
-                prefabEntity.ChildEntities = ConvertComponentPrefabsToEntities(prefab.Children, prefabEntity, deterministicBatchGenerator);
+                // Checkes if the current object being setup is a Placeholder object.
+                if (prefab.Name.Contains("(Placeholder)"))
```

Comment:
I would add a comment here saying that this is a safe method as you checked that the placeholders are all named like this.

Reply:
2cd9536